overlays/custom-packages/kernel/default.nix

```diff
@@ -0,0 +1,6 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+(_final: prev: {
+  makeModulesClosure = x:
```

Comment:
Could this stuff only be enabled when this kernel config stuff is enabled? I mean, this stuff could be moved to `modules/host/kernel/default.nix`

Reply:
Updated as per review comment.